recipes/openpiv/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.20" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alexlib/openpiv-python.git
```

Comment:
We prefer not to clone from version control. Please use a tarball instead and add a checksum (preferably `sha256`) like so.

```
  url: https://github.com/alexlib/openpiv-python/archive/{{ version }}.tar.gz
  sha256: <checksum>
```